.changeset/rule-severity-overhaul.md

```diff
@@ -0,0 +1,19 @@
+---
+"@biomejs/biome": major
+---
+
+Update the default severity level of lint rules.
+
+Every diagnostic emitted by Biome has a severity level set to `error`, `warn`, or `info`.
+Previously, all recommended lint rules had a default severity level set to `error`.
+All other lint rules had a default severity level set to `warn`.
+
+We have adjusted the default severity level of every rule, whether recommended or not, to better communicate the _severity_ that a diagnostic highlights.
+
+- Rules that report hard errors, highly likely erroneous code, dangerous code, or accessibility issues now have a default severity level of `error`.
+- Rules that report possibly erroneous code, or code that could be cleaner when rewrote another way now have a default severity level of `warn`.
+- Rules that reports stylistic suggestions now have a default severity level of `info`.
```

Comment:
That's really a good description that you have here. What do you think about having the same in our documentation **and** the contribution guide?

Reply:
I will add that to the contributor guide. Where do you think we could add this in our documentation?

Reply:
A couple of ideas:
- when we talk about rule severity https://next.biomejs.dev/linter/#rule-severity
- or here https://next.biomejs.dev/linter/#rules, after the first shell command, we mention their default severity, so we can add it there 

I prefer the second option, because the second one is where we _configure_ the severity